src/avram/schema_enforcer.cr

```diff
@@ -180,14 +182,16 @@ module Avram::SchemaEnforcer
 
         Or, you can skip schema checks for this model:
 
-            class #{model_class.to_s} < BaseModel
+            class #{model_class} < BaseModel
               # Great for models used in migrations, or for legacy schemas
               skip_schema_enforcer
             end
 
 
         TEXT
       end
+
+      message
```

Comment:
Is this added for clarity?

Reply:
Yes, I think it looks more obvious than a 30-line if/else block.